src/kbmod/reprojection.py

```diff
@@ -40,11 +40,24 @@ def reproject_image(image, original_wcs, common_wcs):
     image_data = CCDData(image, unit="adu")
     image_data.wcs = original_wcs
 
-    new_image, footprint = reproject.reproject_adaptive(
-        image_data, common_wcs, shape_out=common_wcs.array_shape, bad_value_mode="ignore"
-    )
+    footprint = np.zeros(common_wcs.array_shape, dtype=np.ubyte)
```

Comment:
The `footprint` output array can be pre-allocated. We do so here with a specific dtype=np.ubyte to save size. Previously the would produce a footprint with dtype=float64.